src/projectname/version.py

```diff
@@ -1,6 +1,6 @@
-"""A single source of truth for the version in a programatically-accessible variable.
+"""A single source of truth for the version in a programmatically-accessible variable.
 
 This must only include a single line: __version__ = 'x.y.z'
 """
 
-__version__ = '1.0.0'
+__version__ = "1.0.0"
```

Comment:
I'm guessing that this won't work with release please without some additional config. Maybe the extra file param?

Reply:
Nevermind! I guess that it works! https://github.com/agrc/python/pull/32/files

Reply:
yeah i never was a big fan of this. how do we test it? @jacobdadams

Reply:
I like the idea of having a single source of truth for the version number that is accessible from the code. But maybe it's not as big of a deal now that we auto-bump. What's an alternative?

Reply:
Yeah, this is an attempt to make the version number accessible by code for logging/reporting purposes (ie, if we get a status email we can know if it's using the most recent version or not). It was added in a world where versions were manually controlled, so auto-bumping definitely changes things.